test/parameter_test.py

```diff
@@ -173,6 +177,9 @@ def test_bool_true(self):
         self.run_locally(['Baz', '--bool'])
         self.assertEqual(Baz._val, True)
 
+    def test_bool_default_true(self):
```

Comment:
Also add a `with_config` decorator, setting the default to false, and check that it ranks stronger than `default=`